pontoon/base/models.py

```diff
@@ -1147,6 +1157,17 @@ class Project(AggregatedStats):
     """,
     )
 
+    VISIBILITY_TYPES = (
+        ("public", "Public"),
+        ("private", "Private"),
+    )
+    visibility = models.CharField(
+        max_length=20,
+        default=VISIBILITY_TYPES[0][0],
+        choices=VISIBILITY_TYPES,
+        help_text="""Project managers can see all projects, the rest of users can view only public projects""",
```

Comment:
Can we make this text future-compatible? It will need to change with the next iteration of the spec implementation, if we keep it like this.

How about something like "Change project visibility between private and public"? If you think that's too vague, I'm also fine with removing the `help_text`.

In any case, you'll also need to update the DB migration.